### PR TITLE
iOS SDK 5.2.0 Release Note

### PR DESCRIPTION
**Pull request by zhaowenzhi-lp**
#### Actions:
* Adding release note for iOS SDK 5.2.0
* Adding a page for logging
* Updating APIs in Messaging API

### CODE REVIEW

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+#### Bugs fixed 
+* Fixed issue where white space between keyboard and message area with SDK embedded in tab bar view controller. 
+* Fixed issue where a text typed by the user is shown on the input line place holder ("Write a message").
+* Fixed issue where typing indicator misaligned in Consumer text field.
+* Fixed issue where CSAT displays no Agent icon where agent icon present for agent.
+* Fixed issue where "didFinishWithExpiredToken" not trigger when using Code Flow.
+* Fixed issue where showConversation always create a new socket even when get out and get in within the app.
+* Fixed issue where the color code of System messages gets reset to default when going back into a conversation.
+* Fixed issue where not using device languge until you log out of the SDK and back in.
```
Comment (dvillacis94) [inline pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md:17]: languge -> language

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* Fixed issue where creation date is not set when upload files.
+* Fixed multiple issues where multiple localization keys are missing.
+* Fixed issue where send button label doesn't auto fit with by width.
+* Fixed issue where photo sharing screen is blocking host app from popping conversation view controller in the navigation view controller stack.
+* Fixed issue where 'scrollToBottomButtonBadgeCornerRadius' is not being reflect on the SDK.
+* Fixed issue where inaccurate unread message count between registering and receiving a push notification.
+* Fixed issue where case sensitive URLs not working.
+
+##### Accessibility related bugs fixed
+* Fixed multiple issues around structured content accessibility with better accessibility labels defined and better scolling behavior within carousel elements.
```
Comment (mkithany-lp) [inline pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md:28]: scrolling

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* Fixed multiple issues around quick reply accessibility with better scolling behavior when quick reply buttons are off the screen.
```
Comment (mkithany-lp) [inline pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md:29]: scrolling typo

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  - This API method uses a threshold mechanism of 10 seconds. If this API is called within 10 seconds from the last request was made, the counter will be returned from SDK cache otherwise a new request to the server will be made to fetch the latest count. 
+  - Two new APIs ([getUnreadMessagesCount](mobile-app-messaging-sdk-for-ios-sdk-apis-messaging-api.html#getunreadmessagescount) and [isRegisteredForPushNotifications](mobile-app-messaging-sdk-for-ios-sdk-apis-messaging-api.html#isregisteredforpushnotifications)) introduced.
+  - Two getUnreadMessagesCount APIs deprecated. For more information, please check [Messaging APIs](mobile-app-messaging-sdk-for-ios-sdk-apis-messaging-api.html) in the Deprecated methods.
+  * **New LPConversationsHistoryStateToDisplay** - Add a new enum value (all) for LPConversationsHistoryStateToDisplay. Heasd over [here](mobile-app-messaging-sdk-for-ios-sdk-apis-control-history-apis.html) to find out more!
+#### Known Issues 
+* **Lag of scroll upon first load**: When configued the scrolling configuration as "ScrollToBottom", there is a lag between shoing the conversation screen and scrolling to the bottom.
```
Comment (mkithany-lp) [inline pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md:54]: "When configured"

Comment (mkithany-lp) [reply pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md:54]: Also,"there is a log between showing" ?

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+### Version 5.2.0
+#### iOS Messaging SDK
+
+**Release Date**: May 27, 2020
+
+##### Environmental Requirements
+The iOS Mobile Messaging SDK version 5.2.0 is supported on iOS versions 11 through 13. It is available on Swift version 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15), and Swift version 5.2.4 (swiftlang-1103.0.32.9 clang-1103.0.32.53)
```
Comment (mkithany-lp) [inline pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md:7]: Do we also need to specify Xcode version ?

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/AdvancedFeatures/Logging.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+- **TRACE** (high-traffic logs; tracks many objects as they move through the SDK)
+- **DEBUG** (details some of the inner workings of the SDK logic and network stack)
+- **INFO** (primarily large business-logic steps as the SDK connects, sends, and receives messages)
+- **WARNING** (things that went slightly wrong, but were recoverable enough that the user should not have noticed)
+- **ERROR** (things that went wrong enough that we either failed or had to resort to a fallback that the user may have noticed)
+- **OFF** (no logging output from LivePerson Mobile SDK's)
+
+As is standard with most logging systems, each of these settings also includes all levels below it. Setting the SDK Logging Level to **OFF** disables SDK logging entirely.  The LivePerson Mobile SDK's will retain up to 100 lines of logs for combined all logging levels regardless of the logging level.  This means that if the logging level is set to "ERROR" for example, you may not be able to retrieve the last 100 logs recorded.    
+
+The default log level is **INFO** in Release builds, and **TRACE** in Debug builds.
```
Comment (mkithany-lp) [inline pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/AdvancedFeatures/Logging.md:28]: I am wondering if the default should indeed be INFO?
Any reason why this is not ERROR? For a bran which is heavy use of Mobile, this INFO might fill up lot of their storage. Is that what they want?

Comment (zhaowenzhi-lp) [reply pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/AdvancedFeatures/Logging.md:28]: @randymclainLP

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md (hunk 5 above)
Comment (zhaowenzhi-lp) [reply pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md:7]: We don't specify the Xcode version here.

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md (hunk 4 above)
Comment (zhaowenzhi-lp) [reply pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md]: updated

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md (hunk 3 above)
Comment (zhaowenzhi-lp) [reply pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md]: updated

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md (hunk 2 above)
Comment (zhaowenzhi-lp) [reply pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md]: updated

File: pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md (hunk 1 above)
Comment (zhaowenzhi-lp) [reply pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md]: updatd

**Pull request merged**